.travis/upload-nightly-wheels.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
What will happen in the case where we don't merge any commits for 24 hours? I guess we'll upload a new nightly wheel from the same commit as before?

Reply:
Yeah, and that shouldn't be a problem. I guess you probably want a failsafe that prevents you from uploading a commit from before the current live wheels.